customer-api/client-server.md

```diff
@@ -605,3 +605,21 @@ No payload.
   }
 }
 ```
+
+
+## REST communication
+
+### Methods
+
+#### Send file
+
+| Property | Value |
+|---|----|
+| Endpoint | `/v3/file` |
+| Method | `POST` |
+| Headers | `Content-Type: multipart/form-data; boundary=<boundary>` |
+| Authorization | Done with cookies. Only existing customer can send a file. |
```

Comment:
Only existing customer can send a file -> Only the existing customers can send files

Reply:
jak wyżej